examples/getting-started-app/getting-started-app-scm-1.rockspec

```diff
@@ -9,7 +9,7 @@ dependencies = {
     'lua >= 5.1',
     'luatest == 0.5.0-1',
```

Comment:
There is no luatest anymore. See https://github.com/tarantool/cartridge-cli/blob/master/cartridge-cli/templates/cartridge/files/app_files.lua#L3